Updated 'replace an encrypted disk' text (master). (#549)
freenas/freenas-docs: userguide/storage.rst

in :ref:`Encryption` **before** attempting to replace the failed
drive. Then, follow the steps 1 and 2 as described above. During step
3, a prompt will appear to input and confirm the passphrase for the

> pool. Enter this information then click :guilabel:`Replace Disk`.

Reviewer: ```suggestion
pool. Enter the passphrase and click :guilabel:`Replace Disk`.
```

> Immediately restore the encryption keys to the pool. **If this

Reviewer: Does this need further explanation or maybe a :ref: link for how to restore the encryption keys?

Reviewer 2: Agreed, make "restore the encryption keys" a link (if possible).

> additional step is not performed before the next reboot, access to the

Reviewer: Suggestion to make this warning a little more specific and stronger:
**WARNING: Access to the pool will be permanently lost unless the encryption keys are restored to the pool before the next system reboot!**
